src/components/ec-blog-item/ec-blog-item.story.js

```diff
@@ -0,0 +1,32 @@
+import EcBlogItem from './ec-blog-item.vue';
+
+export default {
+  title: 'Blog Item',
+  component: EcBlogItem,
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: {
+    EcBlogItem,
```

Comment:
Are you using the component anywhere in this file, or just replicating the markup?